### PR TITLE
Clean up en leaks ignore list

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Resolves: OKTA-404422

## Description:

Clean up en leaks ignore list

## PR Checklist

- [x] Have you verified the basic functionality for this change?

### Reviewers:

@jmelberg-okta @aarongranick-okta 

### Issue:

- [OKTA-404422](https://oktainc.atlassian.net/browse/OKTA-404422)

### CODE REVIEW

File: src/v2/view-builder/components/AuthenticatorEnrollOptions.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const AuthenticatorRow = View.extend({
         <div class="authenticator-description">\
           <h3 class="authenticator-label no-translate">{{label}}</h3>\
           {{#if description}}\
-            <p class="authenticator-description--text">{{description}}</p>\
+            <p class="authenticator-description--text no-translate">{{description}}</p>\
```
Comment (jmelberg-okta) [inline src/v2/view-builder/components/AuthenticatorEnrollOptions.js:27]: Shouldn't this `description` point to what is defined [here](https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/AuthenticatorUtil.js#L75-L77)? Seems like we **always** want these to be localized.

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/components/AuthenticatorEnrollOptions.js:27]: yeah, it is not just the authenticator description. In some cases we also have a phone number which is what I was trying to ignore in this case.

Screen Shot 2021-06-15 at 12 16 39 PM

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/components/AuthenticatorEnrollOptions.js:27]: Its for the phone number cases https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/utils/AuthenticatorUtil.js#L55

Comment (nikhilvenkatraman-okta) [reply src/v2/view-builder/components/AuthenticatorEnrollOptions.js]: @jmelberg-okta updated it to add the no-translate className conditionally.

File: src/v2/view-builder/utils/AuthenticatorUtil.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ const getAuthenticatorData = function(authenticator, isVerifyAuthenticator) {
         ? authenticator.relatesTo?.profile?.phoneNumber
         : loc('oie.phone.authenticator.description', 'login'),
       iconClassName: 'mfa-okta-phone',
+      noTranslateClassName: isVerifyAuthenticator? 'no-translate': '',
```
Comment (jmelberg-okta) [inline src/v2/view-builder/utils/AuthenticatorUtil.js:59]: style-nit:

```suggestion
      noTranslateClassName: isVerifyAuthenticator ? 'no-translate' : '',
```

**Review by jmelberg-okta — APPROVED** [review]

**Pull request closed without merging**